drivers/wifi/infineon/cybsp.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <cy_result.h>
+#include <cyhal_sdio.h>
+#include <zephyr/device.h>
+#include <zephyr/dt-bindings/pinctrl/ifx_cat1-pinctrl.h>
+
+/* WIFI interface types */
+#define CYBSP_SDIO_INTERFACE             (0)
+#define CYBSP_SPI_INTERFACE              (1)
+#define CYBSP_M2M_INTERFACE              (2)
```

Comment:
where is eg SPI used?

Reply:
SPI does not support in current version. i will remove CYBSP_SPI_INTERFACE/CYBSP_M2M_INTERFACE